### DISCUSSION TITLE
Are triggers not supported in Strapi?

### DISCUSSION THREAD

**Discussion by mdhameed** [strapi/strapi#7832] [Legacy Topics]
Added a trigger in sqlite. But the trigger is not triggered in strapi when a new content type is added. Though the same trigger works when ran from dbeaver tool.

**Comment by derrickmehaffy** [comment]
This is largely outside Strapi's control, you will want to refer to knex/bookshelf to see why the triggers aren't happening in your case

**Reply by mdhameed** [reply]
> This is largely outside Strapi's control, you will want to refer to knex/bookshelf to see why the triggers aren't happening in your case

Thanks for your feedback. So in general triggers supported in strapi or not? Also, I believe that webhooks can be used as an alternative for my requirement. My requirement is actually, we want to see the Relational field in the contents listview in strapi but this is not currently possible in strapi; only the normal fields are shown. So as a workaround, we thought of denormalizing the table by adding the extra column, this column will be populated once a new content is added. For example, we have 2 contents: Employee and Department, departmentID is FK to employee; so we will add deptName to employee table. This deptName will be updated so that finally we can see the Employees data table with deptName in strapi. Please add your valuable inputs whether this can be achieved via webhooks concept in strapi? Thanks again,

**Reply by derrickmehaffy** [reply]
Triggers are not directly supported by Strapi, no.

We are actively working on supporting relational entires in the list-view so at the moment I don't have a good suggestion for you (as it's something even now, we are exploring internally)

**Reply by mdhameed** [reply]
> Triggers are not directly supported by Strapi, no.
> 
> We are actively working on supporting relational entires in the list-view so at the moment I don't have a good suggestion for you (as it's something even now, we are exploring internally)

ok, thanks. For now, we may go with webhook.
